hourglass.py

```diff
@@ -33,9 +36,29 @@ def build_model(self, mobile=False, show=False):
             self.model.summary()
         
     def train(self, batch_size, model_path, epochs):
-        # TODO
-        pass
+        train_generator = DataGenerator(DEFAULT_TRAIN_ANNOT_PATH, DEFAULT_TRAIN_IMG_PATH, self.inres, self.outres, self.num_stacks, shuffle=True, batch_size=batch_size)
+        val_generator = DataGenerator(DEFAULT_VAL_ANNOT_PATH, DEFAULT_VAL_IMG_PATH, self.inres, self.outres, self.num_stacks, shuffle=True, batch_size=batch_size)
```

Comment:
`DEFAULT_TRAIN_ANNOT_PATH` should be the path to the CSV of annotations you want (generated from a notebook or something). Not the path the JSON annotations.